duckduckgo/iOS
# Remove fire button animation experiment

Task/Issue URL: https://app.asana.com/0/1203936086921904/1204439316428620/f

**Description**:
Remove changes introduced in https://github.com/duckduckgo/iOS/pull/1566

**Steps to test this PR**:
1. Test fire button use
2. Check fire button placement when opening/closing tab switcher (confirm if it is in the same place and no visual glitch occurs)

**Device Testing**:

**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
